src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,120 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[CAPACITY];
+
+    }
```

Comment:
let's remove this black line
```suggestion
        table = new Node[CAPACITY];
    }
```